editor/selectors.js

```diff
@@ -225,6 +225,16 @@ export function getCurrentPost( state ) {
 	return state.currentPost;
 }
 
+/**
+ * Returns the author id of the post currently being edited
+ *
+ * @param  {Object} state Global application state
+ * @return {Number}      ID of post author
+ */
+export function getCurrentPostAuthor( state ) {
```

Comment:
Opens a question as to whether we want granular selectors for every single property of a post, or if it's reasonable to expect a developer to use `getCurrentPost( state ).author` directory (or alternatively, a `getCurrentPostAttribute( state, key )` selector?).